docs/source/markdown/podman-container-clone.1.md.in

```diff
@@ -31,16 +31,7 @@ If none is specified, the original container's CPU runtime period is used.
 
 @@option cpu-shares
 
-If none are specified, the original container's CPU shares are used.
```

Comment:
You can't remove this

Reply:
didn't realize i could leave lines in conjunction w/ the @@options, will add back